cloud-etl/docs/index.rst

```diff
@@ -354,25 +354,40 @@ ksqlDB
    Your output should resemble:
 
    .. code:: bash
-
       [
         {
           "@type": "queries",
           "statementText": "SHOW QUERIES;",
           "queries": [
             {
+              "queryString": "CREATE TABLE COUNT_PER_SOURCE WITH (KAFKA_TOPIC='COUNT_PER_SOURCE', PARTITIONS=6, REPLICAS=3) AS SELECT\n  EVENTLOGS.EVENTSOURCEIP EVENTSOURCEIP,\n  COUNT(*) COUNT\nFROM EVENTLOGS EVENTLOGS\nGROUP BY EVENTLOGS.EVENTSOURCEIP\nEMIT CHANGES;",
```

Comment:
@rspurgeon should this queryString exactly match the query in `statements.sql`?  Same question for below.

Reply:
This is copied directly from the HTTP returned value from ksqlDB server.  IIUC, ksqlDB resolves the query you submit into something more explicit, and the value you get here won't necessarily match what you submit.  I double checked this with the updated statement in the latest commit as well as in the ksqlDB editor in Confluent Cloud.